lib/global-styles.php

```diff
@@ -269,19 +284,20 @@ function gutenberg_experimental_global_styles_get_theme() {
  * @return array Style features supported by the block.
  */
 function gutenberg_experimental_global_styles_get_supported_styles( $supports ) {
-	$style_features = array(
-		'--wp--style--color--link' => array( '__experimentalColor', 'linkColor' ),
-		'background-color'         => array( '__experimentalColor' ),
-		'background'               => array( '__experimentalColor', 'gradients' ),
-		'block-align'              => array( 'align' ),
-		'color'                    => array( '__experimentalColor' ),
-		'font-size'                => array( '__experimentalFontSize' ),
-		'line-height'              => array( '__experimentalLineHeight' ),
+	$mappings = array_merge(
+		gutenberg_experimental_global_styles_get_mappings(),
+		// TODO: remove from here and access directly from lib/blocks.php
```

Comment:
Note that the `block-align` property was added https://github.com/WordPress/gutenberg/commit/0d08cdab0c2aed8611ad4468b86e29247590c60b I think we can fix this in a subsequent PR: it should be removed from here and `lib/blocks.php` refactored to use the block registry instead of global-styles.php